### PR TITLE
Refactor app common

### PR DESCRIPTION
**Pull request by tvlkmirzawidihananta**
This PR is intended to wrap up the common function (helper) in the `app_common.py`, so that the developer can focus on serving the model in the `app.py`
Also in this PR, I propose to format the service output as JSON, since usually it's the preferred format of the client. WDYT?

Best regards 🙇

### CODE REVIEW

File: {{ cookiecutter.app_name }}-serving/src/app.py [hunk 1]
```diff
@@ -1,13 +1,18 @@
-import os
+"""
+Main file to define the prediction flow of the ML model.
+Modify this file as you see fit.
+Refer to this link for the documentation:
+https://29022131.atlassian.net/wiki/spaces/DP/pages/1032225374/Model+Deployment
+"""
 
 from flask import Flask, request
```
Comment (eduardchai) [inline {{ cookiecutter.app_name }}-serving/src/app.py]: This `import Flask` can be removed

**Comment by eduardchai** [comment]
Please ignore the `LGTM` in previous comment. Seems like i typed it accidentally.

Overall, i am okay with the idea. It is felt over-refactoring though especially when the service is small (like this boilerplate). But i can see how separating the common one into another file will help in complex application.

Do you think we should put complex application use case as boilerplate? I'm leaning toward simpler boilerplate. As it gets complicated, it's up to the app owner to refactor it further.

**Comment by tvlkmirzawidihananta** [comment]
current boilerplate is good
IMHO simpler is better 😬

**Comment by tvlkmirzawidihananta** [comment]
too old 🧓

**Pull request closed without merging**